client/interfaces/IHeroCreateOptions.ts

```diff
@@ -7,5 +7,6 @@ export default interface IHeroCreateOptions
     Omit<ISessionCreateOptions, 'scriptInstanceMeta' | 'sessionName' | 'dependencyMap'>
   > {
   name?: string;
+  databox?: any; // this is only here to stop typescript from throwing error
```

Comment:
We should see if we can add this with a module extension in databox (like what the plugins are supposed to be doing). I think it should work since this is an interface

Reply:
In this PR?

Reply:
Just in general. Doesn't have to be in pr